template/src/core/network/githubApi.ts

```diff
@@ -0,0 +1,11 @@
+import Fetcher from "./Fetcher";
+import {fetchJSONMiddleware} from "./fetcherUtils";
+
+const GithubApi = new Fetcher({
+  baseUrl: "https://api.github.com/",
+  initOptions: {headers: {"Content-Type": "application/json"}},
+  responseMiddlewares: [fetchJSONMiddleware],
+  bodyParser: JSON.stringify
+});

```

Comment:
I don't think this should be within the `/src/core/`. We can may be define a `/src/examples` and add these there. They can simple remove that directory after they create a new project